mapping/field.go

```diff
@@ -28,6 +28,7 @@ import (
 var (
 	IndexDynamic = true
 	StoreDynamic = true
+	DocValues    = true // TODO revisit default?
```

Comment:
Shouldn't this be DocValuesDynamic?

Reply:
It was "DocValuesDynamic" in last PR , but you had a comment there suggesting that everywhere we move it to "DocValues". At least that is what I understood there..
Yes, I prefer `DocValuesDynamic`.

Reply:
There is some confusion then.  In the previous version some of them had "Include" and that is what I wanted removed.